test/module/shared_model/validators/batch_validator_test.cpp

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "validators/transaction_batch_validator.hpp"
+
+// TODO igor-egorov 23.04.2019 IR-454 Create tests for batch validator
```

Comment:
Add a description for the tests which are assumed here.

Reply:
Probably good point is to add a stateless validation in `Simulator` class. For this PR is out of the scope, but TODO is ok.

Reply:
@muratovv why is stateless validation required in `Simulator` when proposals are validated in `OrderingGate` when they are received from transport?

Reply:
Blocks stateless validation was added to block factory.